Tests/Microsoft.AppCenter.Test.Windows/AppCenterTest.cs

```diff
@@ -515,10 +515,68 @@ public void ParseAppSecretOnePlatformTerminatingSemicolon()
             var appSecret = Guid.NewGuid().ToString();
             var platformId = "uwp";
             var secrets = $"{platformId}={appSecret};";
-            var parsedSecret = AppCenter.GetSecretForPlatform(secrets, platformId);
+            var parsedSecret = AppCenter.GetSecretAndTargetForPlatform(secrets, platformId);
             Assert.AreEqual(appSecret, parsedSecret);
         }
 
+        /// <summary>
+        /// Verify parse when there is only one platform and both app secret and token
+        /// </summary>
+        [TestMethod]
+        public void ParseAppSecretAndTargetOnePlatformTerminatingSemicolon()
+        {
+            var appSecret = Guid.NewGuid().ToString();
+            var targetToken = Guid.NewGuid().ToString();
+            var platformId = "ios";
+            var secrets = $"{platformId}={appSecret};{platformId}Target={targetToken}";
+            var parsedSecret = AppCenter.GetSecretAndTargetForPlatform(secrets, platformId);
+            var expected = $"appSecret={appSecret};target={targetToken}";
+            Assert.AreEqual(expected, parsedSecret);
+        }
+
+        /// <summary>
+        /// Verify parse when there is several platforms and both app secret and token
+        /// </summary>
+        [TestMethod]
+        public void ParseAppSecretAndTargetMultiplePlatformTerminatingSemicolon()
```

Comment:
The secret string doesn't contain a terminating semicolon.

```suggestion
        public void ParseAppSecretAndTargetMultiplePlatform()
```